docs/pages/database-access/reference/cli.mdx

```diff
@@ -136,7 +136,9 @@ $ teleport db configure bootstrap -c /etc/teleport.yaml --manual
 
 When invoked with a `--format=db` (or `--format=mongodb` for MongoDB) flag,
 produces a CA certificate, a client certificate and a private key file used for
-configuring Database Access with self-hosted database instances.
+configuring Database Access with self-hosted database instances. Note that for database
+formats, tctl must be run on the auth server or the remote
+user must be be able to impersonate the database service.
```

Comment:
I'd consider putting this in an `<Admonition type="note">`

Also "the remote user must be be able to impersonate the database service" doesn't help me much, even as a teleport dev. I think our docs on impersonating services are basically non-existent, but you could at least say "must be able to impersonate the builtin role `Db`, see the impersonation guide (link) for details on how to allow impersonation" (I'm not sure if that role name is right or if they need a user as well)

Reply:
Added